DXMainClient/Online/Connection.cs

```diff
@@ -220,64 +218,49 @@ private void ConnectToServer()
             connectionManager.OnConnectAttemptFailed();
         }
 
-        private void HandleComm(object client)
+        private void HandleComm()
```

Comment:
I agree we should be using the Task based methods. But I didn't want to put too much stuff in this PR since it's not really needed to support .NET6 nor to support cross platform compatibility.
That being said, I already implemented this here:
- https://github.com/Rans4ckeR/xna-cncnet-client/pull/11